guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -31,6 +31,11 @@ ifdef::satellite[]
 include::topics/upgrading_a_disconnected_satellite.adoc[leveloffset=+3]
 endif::[]
 
+ifdef::satellite[]
+// Migrating to a Newly Deployed Satellite Server
+include::modules/proc_migrating_to_a_new_server.adoc[leveloffset=+2]
```

Comment:
Hi @sabuchan07, I think you might want to change file name in this include statement to make your build checks pass:

 ```suggestion
include::modules/proc_migrating-to-a-new-server.adoc[leveloffset=+2]
```